scripts/deploy_to_dist.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Repo to deploy to:
+USER="Yoast-dist"
+REPO="wordpress-seo"
+REPO_URL="git@github.com:$USER/$REPO.git"
+# Get the latest tag
+lastTag=$(git describe --abbrev=0 --tags)
+mainDir=$(pwd)
+# Create a new git repos
+cd ./artifact
+git init
+git remote add origin ${REPO_URL}
+#commit the files
```

Comment:
This file can be cleaned up a bit. Let's use capital letters and full sentences in all comments. Additionally please add some whitespace so it becomes easier to read.